resources/schema/utils.py

```diff
@@ -19,9 +19,7 @@ def resolve_piers(info, **kwargs):
     # Filter out piers with no berths that fit the
     # passed dimensions only if the dimensions were given.
     # Otherwise, return the whole list of piers.
-    has_dimensions_filter = any(
-        ["min_berth_width" in kwargs, "min_berth_length" in kwargs]
-    )
+    has_dimensions_filter = min_width is not None or min_length is not None
```

Comment:
This could be simplified a bit:
```suggestion
    has_dimensions_filter = min_width or min_length
```

Reply:
What if `min_widht` and/or `min_length` is 0?

Reply:
Filtering with 0 is the same as not filtering ¯\_ (ツ)_/¯

Reply:
ok, I changed the handling of None/zero.